capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -58,4 +65,181 @@ public static JsonObject createRequestBodyJson(HttpServletRequest request) {
     }
     return jsonObject;
   }
+
```

Comment:
The static method ```createRequestBodyJson``` (Line 46-67) can be simplified to two lines. It also contains deprecated code. 


#### For example
```
String googleUserTokenObjString = request.getReader().lines().collect(Collectors.joining());
JsonObject googleUserTokenObjJSON = JsonParser.parseString(googleUserTokenObjString).getAsJsonObject();
```

Reply:
Cool! Forgot to change the JsonParser to utilize the newer static methods, but this is a great solution!